lib/email/receiver.rb

```diff
@@ -570,6 +570,14 @@ def parse_from_field(mail = nil)
 
       return unless mail[:from]
 
+      if mail[:reply_to] && mail[:reply_to].errors.blank?
+        mail[:reply_to].each do |address_field|
+          from_address = address_field.address
+          from_display_name = address_field.display_name.try(:to_s)
```

Comment:
Do you need to try here? `nil.to_s` should be fine

Reply:
```suggestion
          from_display_name = address_field.display_name&.to_s
```